tools/config.ts

```diff
@@ -59,7 +60,8 @@ export const NPM_DEPENDENCIES = [
 
 // Declare local files that needs to be injected
 export const APP_ASSETS = [
-  { src: `${ASSETS_SRC}/main.css`, inject: true, dest: CSS_DEST }
+  // inject minified css files from tmp
+  { src: `${TMP_ASSETS_DIR}/main.css`, inject: true, dest: CSS_DEST }
```

Comment:
Is this change required given [`mapPath`](https://github.com/mgechev/angular2-seed/blob/master/tools/tasks/build.index.ts#L30-L33).